src/bootstrap/src/core/sanity.rs

```diff
@@ -35,6 +36,10 @@ const STAGE0_MISSING_TARGETS: &[&str] = &[
     // just a dummy comment so the list doesn't get onelined
 ];
 
+/// Minimum version threshold for libstdc++ required when using prebuilt LLVM
+/// from CI (with`llvm.download-ci-llvm` option).
+const LIBSTDCXX_MIN_VERSION_THRESHOLD: usize = 8;
```

Comment:
Is *minimum* really the right thing here? I'd expect that we need an exact match?

Reply:
Unless there is a breaking change, different versions can be compatible (just like libc). It doesn't need to be the same version.